source/base/process_grid.cc

```diff
@@ -48,32 +49,32 @@ namespace
 
     // Below we always try to create 2D processor grids:
 
-    int n_processes;
+    dealii::types::blas_int n_processes;
```

Comment:
that actually should not be changed, sorry I mislead you a bit by saying that one can change **all* `int` to `dealii::types::blas_int`, have not thought about those things:

```
int MPI_Comm_size( MPI_Comm comm, int *size ) 
```